pkgs/by-name/ma/masterpdfeditor/package.nix

```diff
@@ -31,6 +31,10 @@ stdenv.mkDerivation rec {
     pkcs11helper
   ];
 
+  qtWrapperArgs = [
+    "--prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ nss ]}"
```

Comment:
We have `autoPatchelfHook` btw, does it work if adding to `buildInputs`? If not, try `runtimeDependencies`? `LD_LIBRARY_PATH` seems not recommended for some reasons